app/models/vim_performance_state.rb

```diff
@@ -162,6 +162,14 @@ def get_assoc(relat, mode = nil)
     ids.nil? ? [] : ids.uniq.sort
   end
 
+  # @param obj object holding vps records
+  # @param timestamp [Time|String] hourly timestamp, prefer Time
+  # @returns [Range] time range
+  def self.get_time_interval(obj, timestamp)
```

Comment:
obj isn't even used now - can we remove it?

Reply:
This method is only called for hourly.
I think it was written to support all `VimPerformanceState`.
But we only store vim performance state values for hours so it is probably moot.

changing